gunrock/applications/color.hxx

```diff
@@ -20,37 +20,33 @@
 namespace gunrock {
 namespace color {
 
-template <typename meta_t>
+template <typename graph_t>
```

Comment:
I like these taking `graph_t` as the template instead of `<vertex_t, weight_t, ...>` -- less places that need to have code changed

Reply:
That's fine, you can do it either way. My philosophy is to "include what you need" rather than passing the whole type. But this works too.